metrics/go_metrics_test.go

```diff
@@ -739,6 +740,38 @@ func TestRecordSyncerSet(t *testing.T) {
 	assert.Equal(t, m.SyncerSetsMeter["foo"][SyncerSetUidCleared].Count(), int64(1))
 }
 
+func TestStoredResponses(t *testing.T) {
```

Comment:
Is there a test case that you can create where only one of the two metrics gets incremented? And is there a test case you can create where neither are incremented?

Reply:
`test case that you can create where only one of the two metrics gets incremented` - I think this is added, this is second test case. 
`where neither are incremented` - with current logic I don't see a way to do this. 
I'll be happy to add other test cases, but I cannot really come up with input data.
I modified a description for second test case.